projects/distributed-tracing/src/shared/components/filter-bar/filter-bar.component.ts

```diff
@@ -35,29 +35,29 @@ import { Filter } from './filter/filter-api';
             [scope]="this.scope"
             (apply)="this.onApply($event)"
             (clear)="this.onClear(filter)"
+            (focusin)="this.isFocused = true"
```

Comment:
can we pull these up rather than putting them on each input? Would a focusin on content for example accomplish the same thing with less repetition?